libs/sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -849,6 +848,19 @@ public void testCreateUserStmtBuilder() {
         printStatement("create user \"Günter\"");
         printStatement("create user root");
         printStatement("create user foo with (password = 'foo')");
+        printStatement("create user foo with password 'foo'");
+    }
+
+    @Test
+    public void test_create_role_statement() {
+        // No option
+        printStatement("create role admin");
+        // Single option
+        printStatement("create role admin with inherit password 'foo'");
```

Comment:
Do we also need to support `create role admin with password ?` ?

Reply:
We agreed to go with create role/user to be identical except fro the default value of LOGIN/NO LOGIN, so I unified the functionality in the parser.